xpk.py

```diff
@@ -123,10 +123,20 @@
                 volumeMounts:
                 - mountPath: /dev/shm
                   name: dshm-2
+                - mountPath: /var/run/docker.sock
```

Comment:
We need to talk about the architecture, this doesn't make sense to me to hard code.

Reply:
+1 to avoiding hardcoding this in the common path for TPUs/ CPUs.

Reply:
It seems like both GPU and Pathway mount `/tmp` folder

https://github.com/google/xpk/blob/main/xpk.py#L325-L327
https://github.com/google/xpk/blob/main/xpk.py#L271-L272

Maybe I should mount `/tmp` folder instead of `/tmp/xla_dump/`?

Reply:
GPU / Pathways has specific workload create files (also a sad state of duplication) but that's why it is ok for those mounts. The issue here is we are mounting something that doesn't always exist.